tests/test_sbh_submissions.py

```diff
@@ -1,123 +1,200 @@
-#           sbolDoc.name = 'rule30_Q0_v' + version
-
-#           sbhQuery.submit_Collection(sbh_connector, sbolDoc, True, 0)
+        sbh.submit_collection(doc, collection_ID, collection_version, collection_name,
+                              collection_description, 0, sub_collection_ID, sub_collection_version,
+                              sub_collection_name, sub_collection_description, overwrite=True)
 
+        collection_identity = '/'.join([SD2Constants.SD2_SERVER, 'user', self.user, collection_ID,
+                                        collection_ID + '_collection', '1'])
+        sub_collection_identity = '/'.join([SD2Constants.SD2_SERVER, 'user', self.user, collection_ID,
+                                            sub_collection_ID, '1'])
 
-# if __name__ == '__main__':
-#   unittest.main()
+        query_result = sbh.query_collection_members(collection_uris=[collection_identity])
+
+        assert (len(query_result[collection_identity]) == 1 and sub_collection_identity
+                in query_result[collection_identity])
+
+    def test_submit_to_collection(self):
+        sbh = SynBioHub(SD2Constants.SD2_STAGING_SERVER, self.user, self.password,
+                        SD2Constants.SD2_STAGING_SERVER + '/sparql', SD2Constants.SD2_SERVER)
+
+        sbol.Config.setOption('sbol_typed_uris', False)
+        sbol.Config.setOption('validate', False)
+        sbol.setHomespace('http://dummy.org')
+
+        member_ID1 = 'foo'
+
+        doc1 = sbol.Document()
+        doc1.componentDefinitions.create(member_ID1)
+
+        collection_ID = 'adapter_test_3'
+        collection_name = 'Adapter Test Collection 3'
+        collection_description = 'This is a test of the upload_sbol module of synbiohub_adapter.'
+        collection_version = '1'
+
+        sbh.submit_collection(doc1, collection_ID, collection_version, collection_name,
+                              collection_description, overwrite=True)
+
+        member_identity1 = '/'.join([SD2Constants.SD2_SERVER, 'user', self.user, collection_ID,
+                                     member_ID1, '1'])
+        collection_identity = '/'.join([SD2Constants.SD2_SERVER, 'user', self.user, collection_ID,
+                                        collection_ID + '_collection', '1'])
+
+        member_ID2 = 'bar'
+
+        doc2 = sbol.Document()
+        doc2.componentDefinitions.create(member_ID2)
+
+        sbh.submit_to_collection([doc2], collection_identity)
+
+        member_identity2 = '/'.join([SD2Constants.SD2_SERVER, 'user', self.user, collection_ID,
+                                     member_ID2, '1'])
+
+        query_result = sbh.query_collection_members(collection_uris=[collection_identity])
+
+        assert (len(query_result[collection_identity]) == 2 and member_identity1
+                in query_result[collection_identity] and member_identity2 in query_result[collection_identity])
+
+    def test_submit_to_sub_collection(self):
```

Comment:
If this test is expected to fail as noted in the description, then it should be marked as an [expected failure](https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures) so that the Jenkins build will pass successfully.

Reply:
Marked test as an expected failure.